src/SamplesApp/UITests.Shared/Windows_UI_Xaml/Performance/Performance_Dopes.xaml.cs

```diff
@@ -0,0 +1,255 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Uno.UI.Samples.Controls;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace UITests.Windows_UI_Xaml.Performance
+{
+	/// <summary>
+	/// An empty page that can be used on its own or navigated to within a Frame.
+	/// </summary>
+	[SampleControlInfo("Performance", "MarkupExtension")]
+	public sealed partial class Performance_Dopes : Page
+	{
+		volatile bool breakTest = false;
+		const int max = 600;
+
+		public Performance_Dopes()
+		{
+			this.InitializeComponent();
+		}
+
+		private void StartTestST()
+		{
+			var rand = new Random(0);
+
+			breakTest = false;
+
+			var width = absolute.ActualWidth;
+			var height = absolute.ActualHeight;
+
+			const int step = 20;
+			var labels = new TextBlock[step * 2];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'labels'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5700024)